tasks/deployment_manager/task.ts

```diff
@@ -48,10 +48,12 @@ async function runMigration<T>(
 
 task('deploy', 'Deploys market')
   .addFlag('simulate', 'only simulates the blockchain effects')
+  .addFlag('noDeploy', 'skip the actual deploy step')
```

Comment:
IMO, it's more intuitive if we create a new task in this file called `verify` that only runs the verify process (the `deploy` task will still call verify as well). Running `deploy` with a `noDeploy` flag feels kind of unintuitive.

Reply:
Yeah sure thing, a separate PR is fine